electron/utils.ts

```diff
@@ -25,13 +25,13 @@ const { showErrorBox, showMessageBox } = dialog
 
 const home = homedir()
 const legendaryConfigPath = `${home}/.config/legendary`
-const heroicFolder = `${home}/.config/heroic/`
+const heroicFolder = process.platform === 'darwin' ? `${home}/Library/Application Support/heroic/` : `${home}/.config/heroic/`
 const heroicConfigPath = `${heroicFolder}config.json`
 const heroicGamesConfigPath = `${heroicFolder}GamesConfig/`
 const heroicToolsPath = `${heroicFolder}tools`
 const userInfo = `${legendaryConfigPath}/user.json`
 const heroicInstallPath = `${home}/Games/Heroic`
-const legendaryBin = fixPathForAsarUnpack(join(__dirname, '/bin/legendary'))
+const legendaryBin = fixPathForAsarUnpack(join(__dirname, '/bin/', process.platform, '/legendary'))
```

Comment:
I'm also thinking that we should not include mac binary on linux build, or linux binary on mac builds. (Also windows maybe later on.) 
So we can reduce app size a little bit.

Reply:
I think with automation we can delete the unnecessary files. So, locally we can have all 3, and on the build script, we delete the others, or maybe instead of delete, just copy the necessary one.

Reply:
That's a great idea, we can implement something like that. Moving all binaries to one folder, then copying only required one to public/bin folder on github actions should do it